### DISCUSSION TITLE
What mitigations may lead to the most efficient risk reduction.

### DISCUSSION THREAD

**Discussion by laurie-tyz** [CERTCC/SSVC#227] [Ideas]
- [ ] Where the appropriate decision is "mitigate" use the connection between types of mitigation actions and the SSVC decision points to recommend heuristics about what mitigations may lead to the most efficient risk reduction. 

(Associated with Issue #46 )

**Comment by j---** [comment]
I think this will be pretty hard to do well, and I'm not sure we can accomplish it for v2.0. Maybe we can mention it in Future work (#38) and leave it for a future version?

**Comment by ahouseholder** [comment]
For housekeeping purposes, I removed this issue from the SSVC v2 milestone for now. We can always add it back if we decide to address it in this round.

**Comment by ahouseholder** [comment]
Converting to discussion as there's no clear direction to take here.
